chroma_core/lib/job.py

```diff
@@ -201,6 +201,34 @@ def invoke_agent(self, host, command, args={}):
             self._log_subprocesses(e.subprocesses)
             raise
 
+    def invoke_rust_local_action(self, command, args={}):
+        """
+        Talks to the iml-action-runner service
+        """
+
+        return invoke_rust_local_action(command, args, self._cancel_event)
+
+    def invoke_rust_local_action_expect_result(self, command, args={}):
+        from chroma_core.services.job_scheduler.agent_rpc import LocalActionException
```

Comment:
Why is this being imported locally?

Reply:
If it's only needed in one particular spot sometimes it's easier to import locally. This is especially the case where circular dependencies happen when importing at the top.

Reply:
I suspect it's either to avoid side-effects from importing too early or to avoid a circular dependency.

The other functions that use these exceptions do the same thing, so I'm basically carrying it forward.